client/python/cryoet_data_portal/tests/test_infra/hasura/migrations/cryoetdataportal/1722547580676_alter_table_public_annotation_authors_add_column_primary_author_status/up.sql

```diff
@@ -0,0 +1,3 @@
+alter table "public"."annotation_authors" add column "primary_author_status" boolean
```

Comment:
I copied this from the corresponding change on the backend repo: https://github.com/chanzuckerberg/cryoet-data-portal-backend/commit/60fe5b436631fbcb594b4bcbe71d1fbee2782c0e